### PR TITLE
Enhance serializers with use.latest.version config

### PR DESCRIPTION
**Pull request by rayokota**
Also add skip.known.types config for Protobuf serializer

### CODE REVIEW

File: tests/schema_registry/test_avro_serializer.py [hunk 1]
```diff
@@ -74,6 +74,29 @@ def test_avro_serializer_config_auto_register_schemas_false(mock_schema_registry
...
+
+    test_serializer("test",
+                    SerializationContext("test-use-latest-version",
+                                         MessageField.KEY))
+
+    register_count = test_client.counter['POST'].get('/subjects/{}/versions'
+                                                     .format(subject), 0)
+    assert register_count == 0
+    # Ensure latest was requested
+    assert test_client.counter['GET'].get('/subjects/{}/versions/latest'.format(subject)) == 1
```
Comment (mhowlett) [inline tests/schema_registry/test_avro_serializer.py:97]: there should really be more test coverage of this stuff.
but i'm ok with the effort/benefit tradeoff.

**Review by mhowlett — APPROVED** [review]
thanks, LGTM.

**Comment by Atheuz** [comment]
Fixes bug where a schema lookup is performed for every ProtobufSerializer call. Made Protobuf and schema registry impractical to use for anything but the smallest producers as it would make the lookup, on my network this would take 0.2 seconds per produce. Adds up quick.

**Comment by mhowlett** [comment]
@Atheuz - to clarify - are you saying this PR fixed such a bug, or the bug remains in 1.8.2?

**Comment by Atheuz** [comment]
@mhowlett this PR fixed the bug and as such is fixed in the 1.8.2 release. It just wasn't stated anywhere here, so I wanted to add a comment.

**Pull request merged**